minari/serialization.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import json
+from typing import Dict, Union
+
+import gymnasium as gym
+import numpy as np
+
+
+def serialize_space(space: gym.spaces.Space, to_string=True) -> Union[Dict, str]:
+    if isinstance(space, gym.spaces.Box):
+        result = {}
+        result["type"] = "box"
```

Comment:
For every space name I would use the capital first letter
(not a whim, but it can be helpful for automatic class search)

Reply:
changed